app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,46 @@
+class RecipesController < ApplicationController
+  def index
+    @user = current_user
+    @recipes = @user.recipes
+  end
+
+  def show
+    @user = current_user
+    @recipe = Recipe.includes(recipe_foods: :food).find(params[:id])
+    @recipe_foods = @recipe.recipe_foods
+  end
```

Comment:
the database table is named as `receipe_foods` , there was a little mistake while creating the database from our end but don't worrk I fixed that while reviewing your code .